### DISCUSSION TITLE
RGB LED Matrix instructions copied to wiki?

### DISCUSSION THREAD

**Discussion by corndog2000** [framewiki/framewiki.net#71] [Talk]
I could set up a page with a mirror of the instructions on my personal website for redundancy. Also, then people wouldn't need to leave the wiki.

**Comment by Morpheus636** [comment]
Go for it!

**Comment by corndog2000** [comment]
What's the best way to add images to the repo. Can the website host videos?

**Comment by Morpheus636** [comment]
I don't believe there's a way to host videos. Upload images to ./assets/ and add them to the page with an `<img>` html tag using a relative path. Ideally name them such that you could tell what they're for without looking at them.
